src/project/views/NewUserView.java

```diff
@@ -46,18 +50,24 @@ public static void newUserView(Stage stage){
 		provinceBox.add("QC");
 		provinceBox.add("SK");
 		provinceBox.add("YT");
+		String province = provinceCombo.getValue().toString();
 		Label postCodeLabel = new Label("Postal Code:");
 		postCodeField = new TextField();
+		String postalCode = postCodeField.getText();
 		Label phoneLabel = new Label("Phone:");
 		phoneField = new TextField();
+		String phone = phoneField.getText();

```

Comment:
Wondering if we can make this into an integer, if not I'll make some updates to the user for a string, but there should be some way that we validate this is actually resembles a phone number. Restricts number of digits or something.